ui/app/AppLayouts/Chat/ChatColumn/ChatComponents/ChatCommandModal.qml

```diff
@@ -44,7 +44,7 @@ ModalPopup {
                 currency: walletModel.defaultCurrency
                 width: stack.width
                 //% "From account"
-                label: qsTrId("from-account")
+                label: root.isRequested ? qsTr("Receive on account") : qsTrId("from-account")
```

Comment:
You'll need to move the `qsTr` and `qstrId` on two lines for the comment to be above